### ISSUE TITLE
nginxLocationConfig not generated correctly

### ISSUE THREAD

**Issue by productiveme** [zodern/meteor-up#1149]
Mup version (`mup --version`): 1.4.6

Mup config

```js
{
  "servers": {
    "one": {
      "host": "1.2.3.4",
      "username": "root",
      "pem": "~/.ssh/pem"
    }
  },
  "app": {
    "name": "my-app",
    "path": "../",
    "servers": {
      "one": {}
    },
    "buildOptions": {
      "serverOnly": true
    },
    "env": {
      "ROOT_URL": "https://subdomain.host.com",
      "MONGO_URL": "mongodb://mongodb:27017/my-app",
      "MONGO_OPLOG_URL": "mongodb://mongodb/local",
      "VIRTUAL_HOST": "subdomain.host.com",
      "HTTPS_METHOD": "noredirect",
      "HTTP_FORWARDED_COUNT": 1
    },
    "docker": {
      "image": "abernix/meteord:node-12-base",
      "stopAppDuringPrepareBundle": true,
      "args": [
        "--link=mongodb:mongodb"
      ]
    },
    "enableUploadProgressBar": true,
    "type": "meteor"
  },
  "mongo": {
    "version": "4.0.0",
    "servers": {
      "one": {}
    }
  },
  "proxy": {
    "domains": "subdomain.host.com",
    "nginxLocationConfig": "./nginxLocation.config",
    "shared": {
      "nginxConfig": "./nginxShared.config"
    }
  }
}
```

Output of command

```bash
$ mup setup && mup reconfig && mup proxy reconfig-shared

Started TaskList: Setup Docker
[(private)] - Setup Docker
[(private)] - Setup Docker: SUCCESS

Started TaskList: Setup Meteor
[(private)] - Setup Environment
[(private)] - Setup Environment: SUCCESS

Started TaskList: Setup proxy
[(private)] - Setup Environment
[(private)] - Setup Environment: SUCCESS
[(private)] - Pushing the Startup Script
[(private)] - Pushing the Startup Script: SUCCESS
[(private)] - Pushing Nginx Config
[(private)] - Pushing Nginx Config: SUCCESS
[(private)] - Cleaning Up SSL Certificates
[(private)] - Cleaning Up SSL Certificates: SUCCESS

Started TaskList: Start proxy
[(private)] - Start proxy
[(private)] - Start proxy: SUCCESS

Started TaskList: Setup Mongo
[(private)] - Setup Environment
[(private)] - Setup Environment: SUCCESS
[(private)] - Copying mongodb.conf
[(private)] - Copying mongodb.conf: SUCCESS

Started TaskList: Start Mongo
[(private)] - Start Mongo
[(private)] - Start Mongo: SUCCESS

Next, you should run:
    mup deploy

Started TaskList: Configuring App
[(private)] - Pushing the Startup Script
[(private)] - Pushing the Startup Script: SUCCESS
[(private)] - Sending Environment Variables
[(private)] - Sending Environment Variables: SUCCESS

Started TaskList: Start Meteor
[(private)] - Start Meteor
[(private)] - Start Meteor: SUCCESS
[(private)] - Verifying Deployment
[(private)] - Verifying Deployment: SUCCESS
The shared settings affect all apps using this reverse proxy.

Started TaskList: Configuring Proxy's Shared Settings
[(private)] - Sending shared variables
[(private)] - Sending shared variables: SUCCESS
[(private)] - Sending proxy environment variables
[(private)] - Sending proxy environment variables: SUCCESS
[(private)] - Sending let's encrypt environment variables
[(private)] - Sending let's encrypt environment variables: SUCCESS
[(private)] - Sending nginx config
[(private)] - Sending nginx config: SUCCESS

Started TaskList: Start proxy
[(private)] - Start proxy
[(private)] - Start proxy: SUCCESS
```

```
# nginxLocation.config
proxy_http_version 1.1;
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Connection "upgrade";
proxy_set_header Host $http_host;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forward-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forward-Proto http;
proxy_set_header X-Nginx-Proxy true;
proxy_redirect off;
```

After running the above and inspecting the `/etc/nginx/vhosts.d/sudomain.host.com_location` file it shows the following which breaks the site:
```
proxy_http_version 1.1;
proxy_set_header Upgrade ;
proxy_set_header Connection "upgrade";
proxy_set_header Host ;
proxy_set_header X-Real-IP ;
proxy_set_header X-Forward-For ;
proxy_set_header X-Forward-Proto http;
proxy_set_header X-Nginx-Proxy true;
proxy_redirect off;
```

It seems there is a bug in the file generation where any $xxx string is cleared.

**Comment by productiveme** [comment]
To fix my site I had to `docker exec -it mup-nginx-proxy bash` and edit the file manually before doing a `mup proxy stop && mup proxy start`

**Comment by zodern** [comment]
This is fixed in [Mup 1.5](https://github.com/zodern/meteor-up/pull/976).
